components/script/script_thread.rs

```diff
@@ -2429,6 +2429,9 @@ impl ScriptThread {
         );
         if load_data.url.as_str() == "about:blank" {
             self.start_page_load_about_blank(new_load, load_data.js_eval_result);
+        } else if load_data.url.as_str() == "about:srcdoc" {
+            // < Q2 > Which parameters should be passed to the following method?
+            self.page_load_about_srcdoc(load_data);
```

Comment:
This is a very similar operation to start_page_load_about_black, so I would expect `new_load` and `load_data.srcdoc`.

Reply:
Done. Please review it.